crates/bevy_ecs/src/change_detection.rs

```diff
@@ -118,6 +121,11 @@ pub trait DetectChangesMut: DetectChanges {
     /// **Note**: This operation cannot be undone.
     fn set_changed(&mut self);
 
+    /// Flags this value as having been added.
```

Comment:
It might be worth noting that calling this method should be uncommon.  Something like:

```suggestion
    /// Flags this value as having been added.
    ///
    /// It is not normally necessary to call this method.
    /// The 'added' tick is set when the value is first added,
    /// and is not normally changed afterwards.
```